content/influxdb/cloud/account-management/limits.md

```diff
@@ -0,0 +1,114 @@
+---
+title: InfluxDB Cloud limits and adjustable quotas
+description: >
+  InfluxDB Cloud has adjustable account limits and hard system limits.
+weight: 110
+menu:
+  influxdb_cloud:
+    parent: Account management
+    name: Adjustable quotas and limits
+products: [cloud]
+---
+
+InfluxDB Cloud has adjustable account quotas per organization and hard global system limits.
+
+{{% warn %}}
+All __rates__ (data-in (writes), queries (reads), and deletes) are accrued within a fixed five-minute window. Once a rate is exceeded an error response is returned until the current five-minute window resets.
+{{% /warn %}}
+
+Review adjustable service quotas and global limits to plan for your bandwidth needs:
+
+- [Adjustable service quotas](#adjustable-service-quotas)
+- [Global limits](#global-limits)
+- [UI Error Messages](#ui-error-messages)
+<!-- - [API Error Responses](#api-error-responses) -->
+
+## Adjustable service quotas
+
+To reduce the chance of unexpected charges and protect the service for all users, InfluxDB Cloud has adjustable service quotas applied per account.
+
+_To request higher service quotas, reach out to [InfluxData Support](https://support.influxdata.com/)._
+
+### Free Plan
+
+- **Data-in**: Rate of 5 MB per 5 minutes (average of 17 kb/s)
+  - Uncompressed bytes of normalized [line protocol](/influxdb/cloud/reference/syntax/line-protocol/)
+- **Read**: Rate of 300 MB per 5 minutes (average of 1000 kb/s)
+  - Uncompressed bytes in HTTP in response payload
```

Comment:
This bit did change as we began reporting actual outgoing bytes, so if a user requests the response be compressed, they will be restricted to 300MB of compressed bytes in the HTTP response payload.
```suggestion
  - Bytes in HTTP response payload
```

Reply:
Thank you @glinton -
I was wondering if the compressed/uncompressed felt extra...and whether leaving it out would lead folks to wonder which.